aws/amazon-chime-sdk-component-library-react
# Add Amazon Chime Echo Reduction for React SDK

**Issue #:** 

**Description of changes:**
Add Amazon Chime Echo Reduction for React SDK

**Testing**
1. Have you successfully run `npm run build:release` locally? yes

2. How did you test these changes?
Testing steps:
- Enable Web Audio. Enable Echo Reduction. Join the meeting
- Select one input audio device, and then click on "Enable voice focus". 
- Expect: ns_es model get download and exactly once.
- demo PR: https://github.com/aws-samples/amazon-chime-sdk/pull/109
Echo Reduction On:
<img width="1440" alt="Screen Shot 2021-12-01 at 9 23 56 PM" src="https://user-images.githubusercontent.com/92006736/144363976-a9181b15-ac4c-4c4f-ad75-3f73caf5e15e.png">
Echo Reduction Off:
<img width="1440" alt="Screen Shot 2021-12-01 at 9 24 13 PM" src="https://user-images.githubusercontent.com/92006736/144363989-bd2385b7-6ce9-46d7-83d4-06762a2e2bbc.png">
Default Voice Focus On:
<img width="1440" alt="Screen Shot 2021-12-06 at 9 51 23 PM" src="https://user-images.githubusercontent.com/92006736/144979558-53801b7a-164e-43b1-a3b5-eff111140d05.png">

3. If you made changes to the component library, have you provided corresponding documentation changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
